src/Meplato.Store2.Tests/TestData/availabilities.get.not_found

```diff
@@ -0,0 +1,17 @@
+HTTP/1.1 200
```

Comment:
```suggestion
HTTP/1.1 404 Not Found
```

Reply:
This currently returns a 200 with the following json Data
```
{
    "kind": "store#availabilities/getResponse",
    "items": null,
    "Error": null
}
```

Reply:
OK, are you sure you want to leave it this way? 🤔 Every other service (`Catalogs`, `Jobs`, `Products`) in the .Net client throws in the `Get` endpoint an execution when the requested item doesn't exist. 
So the `Availabilities` service behaves differently, which doesn't feel good TBH, it's not consistent.